src/api/process-manager.ts

```diff
@@ -170,8 +170,9 @@ export function startBuild(data: any): Promise<any> {
 
 export function startJob(proc: JobProcess): Promise<void> {
   return Promise.resolve()
-    .then(() => {
-      startBuildProcess(proc, 'abstruse')
+    .then(() => getRepositoryByBuildId(proc.build_id))
+    .then(repo => {
+      startBuildProcess(proc, 'abstruse', repo.variables.map(v => `${v.name}=${v.value}`))
```

Comment:
this actually works but please move `repo.variables.map(v => `${v.name}=${v.value}`)` to variable and pass that to function.